<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDiskSelectUIClass</class>
 <widget class="QWidget" name="CDiskSelectUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDiskSelectUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_Old">
     <property name="text">
      <string>绑定原SD卡</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_New">
     <property name="text">
      <string>绑定新SD卡</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox_Old"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox_New"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_Old">
     <property name="text">
      <string>原卡:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_New">
     <property name="text">
      <string>新卡:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
